jbpm-services/jbpm-services-api/pom.xml

```diff
@@ -60,6 +60,7 @@
             </configurationFile>
           </analysisConfigurationFiles>
           <reportSeverity>nonBreaking</reportSeverity>
+          <failSeverity>potentiallyBreaking</failSeverity>
```

Comment:
Is this something we could move to the kie-parent? Or do we expect to have different failSeverity in different modules (and similarly for other config options).